### ISSUE TITLE
Run npm audit and fix critical/ high

### ISSUE THREAD

**Issue by wsmith-nwfsc** [nwfsc-fram/boatnet#9]
Currently doing this for obs-electron.

It makes sense to do this _after_ doing the `npm outdated`  and `npm update` commands in ticket #7

To fix the security warnings that node spits out after an `npm install`, run `npm audit` and execute the suggested update commands to fix any critical or high (optionally Medium or Low if you want) warnings.  . I'd avoid updating Low priority issues with a `SEMVER WARNING: Breaking Changes` warning.

There's an `npm audit fix` command which is supposed to do this automatically, but it's not working for me, so I did it manually.

**Comment by wsmith-nwfsc** [comment]
I'll leave this one open, unless you've already done this @sethgerou-noaa (feel free to close)

**Comment by toddhay-NOAA** [comment]
@wsmith-nwfsc and @melinashak-noaa,  please try again and see if you can now see the package security vulnerabilities here:  https://github.com/nwfsc-fram/boatnet/network/alerts

Our data team should now have access to those.

@melinashak-noaa, I'm reassigning this task to you.  Please go through and update our library dependencies to use the latest versions, to fix these vulnerability alerts.

**Comment by sethgerou-noaa** [comment]
updated obs-web - resolved all vulnerabilities

**Comment by wsmith-nwfsc** [comment]
Reopening while Melina performs this for the few remaining repositories.

**Comment by toddhay-NOAA** [comment]
@melinashak-noaa, it looks like there is still a cryptiles issue in survey/surv-cutter

**Comment by wsmith-nwfsc** [comment]
One note: the cryptiles update should be in _**both**_ package.json and package-lock.json.  Since package-lock.json is auto-generated, the cryptiles version could be overwritten unless it's specified in package.json. 
For an example of this, I did that in the obs-electron package.json. I added the line:
`"cryptiles": ">=4.1.2",`
and then did a `npm install` which auto-updates the package-lock.json. Then, committed both files.
